### DISCUSSION TITLE
Livewire is still using `wire:navigate` even when it has been removed from the link

### DISCUSSION THREAD

**Discussion by binaryfire** [livewire/livewire#7767] [1. Help]
Hi all

I'm trying to dynamically disable `wire:navigate`, however when I remove the attribute from an `<a>` tag using js, clicking on the link still ends up navigating. Any ideas why that might be happening and how I could get around it?

Cheers

**Comment by SajadSeyedzadeh** [comment]
Hi, I have the same problem.
Did you find solution?

**Reply by lrljoe** [reply]
Why remove the attribute via JS rather than at blade initiation?

Wire Navigate scans the page at initialisation for elements with wire:navigate.
Removing the attribute won't stop it after that

You will need to unbind the listener

**Reply by binaryfire** [reply]
@lrljoe I was exploring this for frontend-based unsaved changes alerts. Frameworks like Vue have navigation guards but until recently there was no Alpine / Livewire equivalent. Cancellable navigation has now been added to Livewire (https://github.com/livewire/livewire/pull/8093) and solves this problem.

**Reply by SajadSeyedzadeh** [reply]
> @lrljoe I was exploring this for frontend-based unsaved changes alerts. Frameworks like Vue have navigation guards but until recently there was no Alpine / Livewire equivalent. Cancellable navigation has now been added to Livewire (#8093) and solves this problem.
Hi, Thank you very much for sharing.
